### ISSUE TITLE
[5.4]npm run dev (the same with other commands) fail in Windows 10 subsystem bash.

### ISSUE THREAD

**Issue by ac1982** [laravel/framework#17620]
- Laravel Version: 5.4.6
- PHP Version:7.1.1
- Database Driver & Version:MySQL 5.7
- Windows 10 V1607 with latest updates.
- Windows Subsystem Bash Ubuntu 14.04.

### Description:
After upgrade to 5.4.6, fresh install project in windows 10 and with node 7.4 works fine. But npm in windows 10 subsystem bash fail with "95% emitting ERROR  Failed to compile with 3 errors"

### Steps To Reproduce:
1. Node 7.4 and npm 4.1.2 in windows 10 subsystem bash.
2. fresh install laravel 5.4.6.(in bash)
3. npm install.(in bash)
4. npm run production.(in bash)
```
npm ERR! Linux 3.4.0+
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "run" "production"
npm ERR! node v7.4.0
npm ERR! npm  v4.1.2
npm ERR! code ELIFECYCLE
npm ERR! @ production: `node node_modules/cross-env/bin/cross-env.js NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the @ production script 'node node_modules/cross-env/bin/cross-env.js NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the  package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node node_modules/cross-env/bin/cross-env.js NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /mnt/c/Users/alexe/PhpstormProjects/fabtek/npm-debug.log
```
Also upload the log in detail.
[npm-debug.txt](https://github.com/laravel/framework/files/736975/npm-debug.txt)

**Comment by royalforge** [comment]
I'm getting the same error message when requiring Bootstrap javascript files from node_modules. It works fine when I don't require anything from the node_modules.

**Comment by lioannou** [comment]
Have you tried using `yarn`?

I use `yarn --no-bin-links` on Windows 10 with no issues.

**Comment by themsaid** [comment]
Please report the issue on https://github.com/JeffreyWay/laravel-mix

**Comment by ac1982** [comment]
I opened a new issue in https://github.com/JeffreyWay/laravel-mix/issues/236

**Comment by stevencook** [comment]
ac1982 - I had the same issue today when running vagrant homestead on Windows 10 and just got it to work.  I think it has something to do with `npm install --no-bin-links` instead of `npm install` on Windows.  I have to run with --no-bin-links since I'm running Vagrant on Windows.  Then when I ran `npm run dev` and piped the output I saw this info

```
ERROR  Failed to compile with 3 errors

 error  in ./resources/assets/sass/app.scss

Module build failed: Error: ENOENT: no such file or directory, scandir '/home/vagrant/Code/relationship-builder-api/node_modules/node-sass/vendor'
```

I checked out node-sass and it was indeed missing the vendor folder.  I deleted node-sass and installed it individually using `npm install node-sass --no-bin-links`.  (If you try to do `npm install node-sass` it errors out saying that it can't make vendor a symlink`.)  Anyway, that put the vendor folder in there, and now `npm run dev` works.
